consensus/istanbul/backend/backend.go

```diff
@@ -524,10 +524,12 @@ func (sb *Backend) RefreshValPeers(valset istanbul.ValidatorSet) {
 
 	// Disconnect all validator peers if this node is not in the valset
 	if _, val := valset.GetByAddress(sb.Address()); val == nil {
+		sb.logger.Info("Validators Election Results: Node OUT ValidatorSet")
```

Comment:
I am a bit concerned of the logging being within `RefreshValPeers`.  It seems like the logging is a bit misaligned with the purpose of the function.

Have you considered moving it to `NewChainHead()` and added it's own `if address in valset else` logic there?

Reply:
sure!
I agree that is not at all related.